File: saunity/src/main/java/tv/superawesome/plugins/publisher/unity/SAUnityAwesomeAds.java
package tv.superawesome.plugins.publisher.unity;

import android.app.Activity;

import tv.superawesome.sdk.publisher.AwesomeAds;

/**
 * Created by gabriel.coman on 13/05/2018.
 */

public class SAUnityAwesomeAds {

    public static void SuperAwesomeUnityAwesomeAdsInit(Activity activity, boolean loggingEnabled) {
        AwesomeAds.init(activity.getApplication(), loggingEnabled);
    }
}

File: superawesome-unity/src/main/java/tv/superawesome/plugins/publisher/unity/SAUnityAwesomeAds.kt
package tv.superawesome.plugins.publisher.unity

import android.app.Activity
import tv.superawesome.sdk.publisher.common.awesomeAds.AwesomeAds
import tv.superawesome.sdk.publisher.common.models.Configuration
import tv.superawesome.sdk.publisher.common.network.Environment

/**
 * Created by gabriel.coman on 13/05/2018.
 */
object SAUnityAwesomeAds {
    fun SuperAwesomeUnityAwesomeAdsInit(activity: Activity, loggingEnabled: Boolean) {
        AwesomeAds.init(activity.application, Configuration(Environment.Production, loggingEnabled))
    }
}